src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -28,17 +52,17 @@
 		<!-- Workbench Icon/Manifest/CSS -->
 		<link rel="icon" href="/favicon.ico" type="image/x-icon" />
 		<link rel="manifest" href="/manifest.json" />
-		<link data-name="vs/workbench/workbench.web.main" rel="stylesheet" href="./static/out/vs/workbench/workbench.web.main.css">
+		<link data-name="vs/workbench/workbench.web.main" rel="stylesheet" href="./static/out/vs/workbench/workbench.web.main.css" onerror="onVsCodeWorkbenchError(event)">
 
 	</head>
 
 	<body aria-label="">
 	</body>
 
 	<!-- Startup (do not modify order of script tags!) -->
-	<script type="text/javascript" src="/_supervisor/frontend/main.js" charset="utf-8"></script>
-	<script src="./static/out/vs/loader.js"></script>
-	<script src="./static/out/vs/webPackagePaths.js"></script>
+	<script type="text/javascript" src="/_supervisor/frontend/main.js" onerror="onVsCodeWorkbenchError(event)" charset="utf-8"></script>
+	<script src="./static/out/vs/loader.js" onerror="onVsCodeWorkbenchError(event)" ></script>
+	<script src="./static/out/vs/webPackagePaths.js" onerror="onVsCodeWorkbenchError(event)" ></script>
 	<script>
 		Object.keys(self.webPackagePaths).map(function (key, index) {
```

Comment:
ideally we need to hook into amd loading as well, since all code for terminals and so on is loaded via it, let's do with another PR

Reply:
actually maybe we don't need to do anything, if it happens after `load` event then supervisor fronted will already handle it